graphql/database/experiments/features.js

```diff
@@ -135,6 +136,66 @@ const features = {
       },
     ],
   },
+  [YAHOO_SEARCH_NEW_USERS_V2]: {
+    defaultValue: 'Tooltip',
+    rules: [
+      /* Begin internal overrides */
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${YAHOO_SEARCH_NEW_USERS_V2}`]: {
+            $eq: 'Control',
+            $exists: true,
+          },
+        },
+        force: 'Control',
+      },
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${YAHOO_SEARCH_NEW_USERS_V2}`]: {
+            $eq: 'Tooltip',
+            $exists: true,
+          },
+        },
+        force: 'Tooltip',
+      },
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${YAHOO_SEARCH_NEW_USERS_V2}`]: {
+            $eq: 'Notification',
+            $exists: true,
+          },
+        },
+        force: 'Notification',
+      },
+      /* End internal overrides */
+      {
+        variations: ['Control', 'Tooltip', 'Notification'],
+        weights: [0.34, 0.33, 0.33],
+        coverage: 1.0,
```

Comment:
As-is, this would start assigning users to the experiment when deployed, right? Set coverage to zero.

Reply:
This should be 0, but just to clarify I intended the `isTabTeamMember` condition to gate that behavior.

Reply:
That's my mistake. I overlooked that condition. 👍